asset/css/variables.less

```diff
@@ -162,5 +181,13 @@
     --suggestions-relation-path-focus-bg: var(--base-gray);
 
     --card-border-color: var(--base-gray-light);
+
+    --schedule-element-fields-bg: var(--default-input-bg);
+    --schedule-element-default-text-color: var(--base-primary-color);
```

Comment:
Interesting. In the dark theme it's initalized with `@default-bg`. Why here not with `var(--default-bg)` as well? I mean, please not with `default-bg` again, but with the same variable counterpart please!

Reply:
I don’t wanna go uneccessarily deep into this, but I’d rather opt for `@default-body-bg` or `@default-viewport-bg` to be less ambiguous for both.